data/transition-sites/slc.yml

```diff
@@ -0,0 +1,6 @@
+---
+site: slc
+whitehall_slug: student-loans-company
+homepage: https://www.gov.uk/government/organisations/student-loans-company
+tna_timestamp:  20201010101010 # Stub timestamp - site not in TNA
```

Comment:
I can see there was [a national archives capture of this website on the 21st December last year](https://webarchive.nationalarchives.gov.uk/20181221130027/https://www.slc.co.uk/) - should we use that as the timestamp?

Reply:
👍 good idea.